index.html

```diff
@@ -9,14 +9,24 @@
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet">
+    <script src="script.js"></script>
 </head>
 <body>
     <div>
         <nav class="hdr">
             <div class="logo">
                 <p>My Logo</p>
             </div>
-            <div class="bar"><img src="./images/Union.png" alt="hamburger menu"></div>
+            <div id="myButtons" class="overlay">
+                <a href="javascript:void(0)" class="closebtn" onclick="closeMenu()">&times;</a>
+                <div class="overlay-content">
+                    <a href="#portfoliostart">Portfolio</a>
+                    <a href="#aboutmelink">About</a>
+                    <a href="#contactmelink">Contact</a>
+                </div>               
+            </div>
+            <span class="hamburger" onclick="openMenu()">&#9776;</span>
```

Comment:
- [ ] The hamburger and it's corresponding  close button are showing on desktop screen. These are supposed to replace the actual menu on mobile screen due to limited space on mobile screen. Kindle ensure to improve upon this aspect. 

**Mobile menu showing on desktop screen:**
![image](https://user-images.githubusercontent.com/12745474/188550847-91e5df50-4136-412a-884c-a023225d358d.png)

Reply:
Thank you, done!